eogrow/pipelines/rasterize.py

```diff
@@ -34,39 +26,11 @@
 LOGGER = logging.getLogger(__name__)
 
 
-class VectorColumnSchema(BaseSchema):
```

Comment:
Wasn't sure about reducing this extra layer of schemas. On one hand it's nicely packed for the raster output, but on the other it's jut an additional complication and layer in the JSON configs, which can look quite messy. It was also easier to validate the fields like this.